<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDD2013WeeklySetupDialog</class>
 <widget class="QDialog" name="ReportDD2013WeeklySetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblTerType">
     <property name="text">
      <string>Территориально</string>
     </property>
     <property name="buddy">
      <cstring>cmbTerType</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblPayStatus">
     <property name="text">
      <string>Состояние оплаты</string>
     </property>
     <property name="buddy">
      <cstring>cmbPayStatus</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cmbSex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>129</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2">
    <widget class="QComboBox" name="cmbSource">
     <item>
      <property name="text">
       <string>База данных</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Предоставленные значения</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="chkCountUnfinished">
     <property name="text">
      <string>Учитывать неоконченные обращения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="19" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QGroupBox" name="grbPopulation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Численность населения</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>340</width>
        <height>93</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QLabel" name="lblWomen">
         <property name="text">
          <string>Всего женщин</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lblWomenPlan">
         <property name="text">
          <string>План</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblYoung">
         <property name="text">
          <string>21-36 лет</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblMenPlan">
         <property name="text">
          <string>План</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblMiddle">
         <property name="text">
          <string>39-60 лет</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblOld">
         <property name="text">
          <string>старше 60 лет</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblMen">
         <property name="text">
          <string>Всего мужчин</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtYoungMen"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtMiddleMen"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edtOldMen"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="edtYoungMenPlan"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="edtMiddleMenPlan"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="edtOldMenPlan"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="edtYoungWomen"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="edtMiddleWomen"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="edtOldWomen"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="edtYoungWomenPlan"/>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="edtMiddleWomenPlan"/>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="edtOldWomenPlan"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="cmbTerType">
     <item>
      <property name="text">
       <string>Не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проживает в субъекте, обслуживаемом ЛПУ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прикреплен к ЛПУ</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QComboBox" name="cmbPayStatus"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс соц. статуса</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="CSocStatusComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Тип соц. статуса</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblSource">
     <property name="text">
      <string>Источник данных о населении</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblDDPlan">
     <property name="text">
      <string>План диспансеризации</string>
     </property>
     <property name="buddy">
      <cstring>edtDDPlan</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="edtDDPlan">
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CSocStatusComboBox</class>
   <extends>QComboBox</extends>
   <header>Registry/SocStatusComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>cmbTerType</tabstop>
  <tabstop>cmbPayStatus</tabstop>
  <tabstop>chkCountUnfinished</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportDD2013WeeklySetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportDD2013WeeklySetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
